Package.swift

```diff
@@ -0,0 +1,9 @@
+import PackageDescription
+
+let package = Package(
+    name: "GRDB.swift",
```

Comment:
OK, you made it clear that it's not quite clear ;-) In doubt let's rename to `GRDB`. This `.swift` suffix can only add confusion or nasty surprises ;-)